app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -3,12 +3,14 @@ class ApplicationFormController < CandidateInterfaceController
     before_action :redirect_to_dashboard_if_not_amendable, only: %i[show review]
     before_action :redirect_to_application_form_unless_submitted, only: %i[review_submitted complete submit_success]
 
+
     def show
       @application_form_presenter = CandidateInterface::ApplicationFormPresenter.new(current_application)
       @application_form = current_application
     end
 
     def review
+      refirect_to_application_dashboard_if_amendable
```

Comment:
I don't think you need this line since you included in ` before_action :redirect_to_dashboard_if_not_amendable, only: %i[show review]`

Reply:
Moved this into an inline to avoid confusion. 👍